airbyte-temporal/README.md

```diff
@@ -1,3 +1,7 @@
+# airbyte-temporal
+
+This module wraps the publicly available Temporal Docker image (temporal.io). It decorates it with functionality that makes it so that users of Airbyte do not need to do anything manual when the Airbyte platform upgrades the version of Temporal that it is using.
```

Comment:
```suggestion
This module wraps the publicly available Temporal Docker image (temporal.io). It decorates it with functionality that makes it so that users of Airbyte do not need to do anything manual when the Airbyte platform upgrades the version of Temporal that it is using.
```

This module implements a custom version of what the temporal autosetup image is doing. It ensure that the temporalDB schema will get upgraded if the temporal version is updated.

Reply:
thanks. added